providers/implementations/ciphers/cipher_aes_wrp.c

```diff
@@ -212,12 +226,12 @@ static int aes_wrap_cipher(void *vctx,
 
     if (outsize < inl) {
         ERR_raise(ERR_LIB_PROV, PROV_R_OUTPUT_BUFFER_TOO_SMALL);
-        return -1;
+        return 0;
     }
 
     len = aes_wrap_cipher_internal(ctx, out, in, inl);
-    if (len == 0)
-        return -1;
+    if (len <= 0)
```

Comment:
Should this be `len < 0` or just `len == -1`? I would argue 0 is a valid return value from `aes_wrap_cipher_internal`, as it means the input/output is empty.

Reply:
Nope:

https://github.com/openssl/openssl/blob/ed3f51ec7d2f8956bff7bedf99b65ff1595e02e8/providers/implementations/ciphers/cipher_aes_wrp.c#L154-L156

Reply:
What about: https://github.com/openssl/openssl/blob/ed3f51ec7d2f8956bff7bedf99b65ff1595e02e8/providers/implementations/ciphers/cipher_aes_wrp.c#L150-L152

Should `aes_wrap_cipher` return an error in that case?